### PR TITLE
Logger context dependant

### LINKED ISSUE #757: Problems when Showing Service Descriptions
## Long story short
There are update issues when selecting services from the list and showing their descriptions. Not updated properly.

## Expected behaviour
That when I click on a service in the list, its appropriate details are shown.

## Actual behaviour
See animation:

It worked quite fine in the beginning ... but then suddenly stuck a bit when selecting different services from the list.

## Steps to reproduce
Please see above.

## Your environment
```json
[
  {
    "name": "osparc-simcore",
    "version": "57c704a",
    "url": "https://github.com/ITISFoundation/osparc-simcore/tree/57c704a/"
  },
  {
    "name": "osparc-simcore UI ['clean']",
    "version": "57c704a",
    "url": "https://github.com/ITISFoundation/osparc-simcore/tree/57c704a/services/web/client/"
  },
  {
    "name": "qooxdoo-compiler",
    "version": "0.3.0-alpha.20190423-1234",
    "url": "https://github.com/qooxdoo/qooxdoo-compiler"
  },
  {
    "name": "qxapp",
    "version": "1.0.0",
    "url": ""
  },
  {
    "name": "qooxdoo framework",
    "version": "6.0.0-alpha-20190409-f79cc78",
    "url": "http://qooxdoo.org"
  },
  {
    "name": "qx-iconfont-fontawesome5",
    "version": "0.0.6",
    "url": "https://github.com/ITISFoundation/qx-iconfont-fontawesome5"
  },
  {
    "name": "qx-iconfont-material",
    "version": "0.1.4",
    "url": "https://github.com/ITISFoundation/qx-osparc-theme"
  },
  {
    "name": "qx-osparc-theme",
    "version": "0.4.6",
    "url": "https://github.com/ITISFoundation/qx-osparc-theme"
  },
  {
    "name": "socket.io",
    "version": "2.1.2",
    "url": "https://github.com/socketio/socket.io"
  },
  {
    "name": "jsondiffpatch",
    "version": "0.3.11",
    "url": "https://github.com/benjamine/jsondiffpatch"
  },
  {
    "name": "jsonTreeViewer",
    "version": "0.6.0",
    "url": "https://github.com/summerstyle/jsonTreeViewer"
  },
  {
    "name": "Ajv",
    "version": "6.5.0",
    "url": "https://github.com/epoberezkin/ajv"
  },
  {
    "name": "plotly",
    "version": "1.43.2",
    "url": "https://github.com/plotly/plotly.js"
  },
  {
    "name": "gridster",
    "version": "0.7.0",
    "url": "https://github.com/dsmorse/gridster.js"
  },
  {
    "name": "svg.js",
    "version": "2.6.4",
    "url": "https://github.com/svgdotjs/svg.js"
  }
]
```

### LINKED ISSUE #761: Make Logger context dependant
Logger should react to context changes.
- When the user is in the main workbench view, all logs should be displayed
- When the user enters a node, only those logs coming from that node should be displayed

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?
- [x] Log message changes when label is changed
- [x] When entering a node, it's label is added to the logger's search filter, showing only the messages coming from that node.
logger

## Related issue number
closes #761

**Bonus:**
closes #757
connects #752 (Brought back the alert window)
show version number in login page:
image

## Checklist

- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] **Runs in the swarm**
- [ ] If you design a new module, add your user to .github/CODEOWNERS

### CODE REVIEW

**Comment by pcrespov** [comment]
Can you please make the footnote with the version much smaller and add organization as well?
Similar to this (but keep it at the bottome of the page):

image


Another nice example are our friends from https://app.blackfynn.io/ . See how they use it to add terms of user etc ... In our case, i would leave there some kind of placeholder to add that in the future

**Comment by odeimaiz** [comment]
> Please consider my notes above

Absolutely, thanks!

File: services/web/client/source/class/qxapp/component/widget/logger/LoggerView.js [hunk 1]
```diff
@@ -57,15 +57,19 @@ Object.freeze(LOG_LEVEL);
 qx.Class.define("qxapp.component.widget.logger.LoggerView", {
   extend: qx.ui.core.Widget,
 
-  construct: function() {
+  construct: function(workbench) {
     this.base();
 
+    this.set({
+      workbench
```
Comment (oetiker) [inline services/web/client/source/class/qxapp/component/widget/logger/LoggerView.js]: odd syntax :) what does this do ?

Comment (odeimaiz) [reply services/web/client/source/class/qxapp/component/widget/logger/LoggerView.js]: it sets the variable workbench to the property with the same name

**Pull request merged**